pkg/storkctl/clusterpair.go

```diff
@@ -430,7 +434,7 @@ func newCreateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptions
 			credentialData["encryptionKey"] = []byte(encryptionKey)
 
 			// Bail out if portworx-api service type is not loadbalancer type and the endpoints are not provided.
-			if len(sEP) == 0 {
+			if !uniDirectional && len(sEP) == 0 {
```

Comment:
I am trying to understand change in this line. Can you please explain?

Reply:
We don't have to figure out portworx endpoint in source cluster for unidirectional clusterpair.
Also with this check the output of unidirectional create command does not print the following kind of line
`Source portworx endpoint is 10.10.101.100:9001`